test/src/unit-cppapi-array.cc

```diff
@@ -563,6 +563,32 @@ TEST_CASE("C++ API: Encrypted array", "[cppapi], [encryption]") {
     vfs.remove_dir(array_name);
 }
 
+TEST_CASE(
+    "C++ API: Open array with anonymous attribute",
+    "[cppapi], [cppapi-open-array-anon-attr]") {
+  Context ctx;
+  VFS vfs(ctx);
+  const std::string array_name = "cppapi_open_array_anon_attr";
+  if (vfs.is_dir(array_name))
+    vfs.remove_dir(array_name);
+
+  // Create array
+  Domain domain(ctx);
+  domain.add_dimension(Dimension::create<int>(ctx, "d", {{1, 4}}, 4));
+  ArraySchema schema(ctx, TILEDB_DENSE);
+  schema.set_domain(domain);
+  schema.add_attribute(Attribute::create<int>(ctx, ""));
+  Array::create(array_name, schema);
+
+  REQUIRE_THROWS_AS(
```

Comment:
Why does opening for write fail here?

Reply:
I believe this signature fails because it passing a timestamp of 0. But this is a copy oversight from the `[cppapi-open-array-at]` test that I started from.

Not really relevant to this attribute test, so I will remove it.